tracker/tracker_debugger_api.yaml

```diff
@@ -61,6 +61,61 @@ paths:
             example: john@doe.com
           in: query
           required: false
+        - name: page_url
+          description: Filters out sessions that don't contain at least one event with URL matching the specified string.
+          schema:
+            type: string
+            example: blog
+          in: query
+          required: false
+        - name: event_type
+          description: |
+            Filters out sessions that don't contain at least one event of the specified type.
+            The allowed values are:
+            - 1 # page view
```

Comment:
Looks better, but do we want to separate number from label with `#`?
![2021-06-30T100434](https://user-images.githubusercontent.com/4488774/123925040-d22fe300-d98a-11eb-8e7a-4a43c2b8c4b2.png)

Reply:
it makes copy-paste easier

Reply:
and about link to JWT, it is a subject for another ticket, this one is already overdue